agoraStreaming/src/Screens/Home/index.tsx

```diff
@@ -72,14 +72,16 @@ export const Home: FC<HomeScreenProps> = () => {
       });
   }, []);
 
-  const choseChannelAndJoinLive = (channelId: string) => {
+  const choseChannelAndJoinLive = (channelId: string, isVideo: boolean) => {
```

Comment:
use type that you already decelerated it in RootStackParamList, like RootStackParamList['Live']['isVideo'] or just create IsVideoLive: boolean and use it everywhere